languages/anaconda3/template/requirements.yaml

```diff
@@ -0,0 +1,14 @@
+channels:
+# Add your desired anaconda channels here
+- conda-forge
+- defaults
+dependencies:
+# Here are your anaconda defined dependencies, including the python version
+# If you wish to use a different python version, please adjust the value below
```

Comment:
doesn't that mean that every algorithm would end up with their own installation of python then? that feels pretty inefficient

Reply:
yes, I believe we can pre-bake a version into the docker image and use that instead if pre-installed though. But this does allow us to have 1 docker image for _any_ python version which I think is pretty versatile!

Reply:
We could also skip that line and use the preinstalled version of python in the image (aka using a python37 base image or w/e along with the anaconda image) and just not provide it as a default customization option.

Reply:
Going to have to agree with Patrick here, we should probably not provide this as a customization option right now. By installing a fresh python version in each algorithm, we essentially invalidate any caching/sharing benefits we receive from Docker.